pipestat_example/.looper.yaml

```diff
@@ -0,0 +1,9 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+pipestat:
+  sample:
+    sample_name: frog_1
+    schema_path: ./pipeline/pipestat_output_schema.yaml
+    results_file_path: "results.yaml"  #this is an absolute path for now
```

Comment:
this comment says this is an absolute path, but it's not.

Reply:
fixed